app/webpack/javascripts/modules/determination.mjs

```diff
@@ -0,0 +1,77 @@
+/**
+ * Determination
+ *
+ * Calculate totals for the assessment summary on case_workers/claims/<claim-id>
+ */
+export class Determination {
+  /**
+   * @param {Element} $module - HTML element to use for component
+   */
+  constructor ($module) {
+    if (($module instanceof window.HTMLElement) && document.body.classList.contains('govuk-frontend-supported')) {
+      this.$module = $module
+    }
+  }
+
+  /**
+   * Initialise component
+   */
+  init () {
+    // Check that required elements are present
+    if (!this.$module) {
+      return
+    }
+
+    const $module = this.$module
+
+    this.$totalExclVat = document.getElementsByClassName('js-total-exc-vat-determination')[0]
+    this.$totalVat = $module.getElementsByClassName('js-vat-determination')[0]
+    this.$LgfsVat = $module.getElementsByClassName('js-lgfs-vat-determination')[0]
+    this.$totalInclVat = $module.getElementsByClassName('js-total-determination')[0]
+    this.scheme = $module.dataset.scheme
+    // Should this be this.applyVat?
+    this.ajaxVat = $module.dataset.applyVat
+    this.vatUrl = $module.dataset.vatUrl
+    this.vatDate = $module.dataset.submittedDate
+
+    this.fields = [
+      document.getElementById('claim_assessment_attributes_fees'),
+      document.getElementById('claim_assessment_attributes_expenses'),
+      document.getElementById('claim_assessment_attributes_disbursements'),
+      document.getElementById('claim_assessment_attributes_vat_amount')
+    ].filter(field => field)
+    this.fields.forEach(element => element.addEventListener('change', () => this.calculateTotalRows()))
+  }
+
+  calculateTotalRows () {
+    const total = this.fields.reduce((n, field) => n + this.parsedValue(field), 0).toFixed(2)
+
+    this.applyVat(total).then(data => {
+      this.$totalExclVat.innerHTML = data.net_amount
+      if (this.$totalVat) { this.$totalVat.innerHTML = data.vat_amount }
+      this.$totalInclVat.innerHTML = data.total_inc_vat
+    })
+  }
+
+  parsedValue (field) {
+    const value = parseFloat(field?.value)
+    if (value) {
+      return Math.max(0.0, value)
+    } else {
+      return 0.0
+    }
+  }
+
```

Comment:
Can be written as : 
`function parsedValue(field) {
  const value = parseFloat(field?.value) || 0.0;
  return Math.max(0.0, value);
}`